src/Bubble.js

```diff
@@ -161,6 +161,21 @@ const styles = {
       borderTopLeftRadius: 3,
     },
   }),
+  center: StyleSheet.create({
+    container: {
+      flex: 1,
+      alignItems: 'center',
+    },
+    wrapper: {
+      borderRadius: 15,
+      backgroundColor: '#7E8B99',
+      marginHorizontal: 30,
+      minHeight: 20,
+      justifyContent: 'flex-end',
+    },
+    containerToNext: {},
+    containerToPrevious: {},
```

Comment:
At first I thought that messages from system could have 'user' field and handleBubbleToNext/Previous would throw an error.
But after your earlier suggestion that messages from system shouldn't have user field they are unnecessary.
I will fix it and test with new react in ~24h